pages/graphql.vue

```diff
@@ -318,79 +318,102 @@
       </div>
 
       <aside class="sticky-inner inner-right lg:max-w-md">
-        <AppSection class="purple" :label="$t('docs')" ref="docs" no-legend>
-          <section class="flex-col">
-            <input
-              type="text"
-              :placeholder="$t('search')"
-              v-model="graphqlFieldsFilterText"
-              class="rounded-t-lg"
-            />
-            <SmartTabs ref="gqlTabs" styles="m-4">
-              <div class="gqlTabs">
-                <SmartTab
-                  v-if="queryFields.length > 0"
-                  :id="'queries'"
-                  :label="$t('queries')"
-                  :selected="true"
-                >
-                  <div v-for="field in filteredQueryFields" :key="field.name">
-                    <GraphqlField :gqlField="field" :jumpTypeCallback="handleJumpToType" />
+        <SmartTabs styles="m-4">
```

Comment:
```suggestion
        <SmartTabs>
```
Doesn't require margin here.